.github/workflows/ci.yml

```diff
@@ -161,14 +167,19 @@ jobs:
             --verbosity 2
           coverage combine
           mv .coverage .coverage-regular
+        env:
+          DJANGO_SETTINGS_MODULE: openzaak.conf.ci
+          SECRET_KEY: dummy
+          DB_USER: postgres
+          DB_PASSWORD: ''
+          DB_POOL_ENABLED: ${{ matrix.use_pooling }}
+          
 
       - name: Upload Coverage Report
-        # TODO find a better way to upload this only once
-        if: ${{ matrix.postgres == '17' && matrix.postgis == '3.5' }}
         uses: actions/upload-artifact@v4
         with:
-          name: coverage-regular-tests
-          path: ${{ github.workspace }}/.coverage-regular
+          name: coverage-regular-tests-pg${{ matrix.postgres }}-postgis${{ matrix.postgis }}${{ matrix.use_pooling && '-pooling' || '' }}
+          path:  ${{ github.workspace }}/.coverage-regular
           include-hidden-files: true
 
```

Comment:
I think the if statement should be kept op only upload the coverage of pg 17 with gis 3.5

To keep the naming consistent i think coverage-regular-tests & coverage-db-pooling-tests is better. The path should also probably change based on `matrix.use_pooling`.